### ISSUE TITLE
enable auto-update for windows

### ISSUE THREAD

**Issue by alfred-landrum** [brimsec/brim#552]
Now that we have code-signing support for our windows builds, we should be able to create the zip needed for the auto-update code.

EDIT: Our reading of the docs says that the exe file we produce should be enough for windows auto-update to work, but it doesn't appear to do so on the recent v0.7.0 release.

**Comment by philrz** [comment]
@mason-fish believes this is due to us not handling our auto-update message in Electron the way we should.

**Comment by philrz** [comment]
Verified in Brim tagged `v0.9.1`. I started out running the short-lived `v0.9.0` release, and after launching and waiting less than a minute, I was prompted.

![image](https://user-images.githubusercontent.com/5934157/81211020-a0d6a580-8f87-11ea-8f45-4886d4493d90.png)

Once I let it restart, I could confirm I was running the new version.

![image](https://user-images.githubusercontent.com/5934157/81211084-b8159300-8f87-11ea-8d52-f9e9c5f2c026.png)
